src/blocks/block-coauthor-avatar/style.css

```diff
@@ -0,0 +1,33 @@
+/* Default Layout */
+
+.wp-block-co-authors-plus-avatar {
+	margin-bottom: 0;
+}
+
+.wp-block-co-authors-plus-avatar :where(img) {
+	height: auto;
+	max-width: 100%;
+	vertical-align: bottom;
+}
+
+/* Inline Layout */
+
+.wp-block-co-authors-plus-coauthors.is-layout-cap-inline .wp-block-co-authors-plus-avatar {
+	display: inline;
```

Comment:
I believe this is already taken care of by the `:where(*)` styles in the parent coauthors block.

Reply:
I agree, I removed that here:
https://github.com/thoughtis/Co-Authors-Plus/commit/972f8729b5a5a5188032a2829f52927b907e9866